elasticsearch/helpers/vectorstore/_async/vectorstore.py

```diff
@@ -60,6 +60,7 @@ def __init__(
         vector_field: str = "vector_field",
         metadata_mappings: Optional[Dict[str, Any]] = None,
         user_agent: str = f"elasticsearch-py-vs/{lib_version}",
+        custom_settings: Optional[Dict[str, Any]] = None,
```

Comment:
```suggestion
        custom_index_settings: Optional[Dict[str, Any]] = None,
```

I think it is best to clarify that these are settings for the index.

Reply:
i see, thx